src/drivers/mct/shr/seq_drydep_mod.F90

```diff
@@ -753,6 +1344,93 @@ subroutine seq_drydep_init( )
              test_name = 'CO'  ! this is just a place holder. values are set in drydep_fromlnd
           case( 'NH4NO3' )
              test_name = 'HNO3'
+          ! Below add GEOS-Chem species or mappings
```

Comment:
Can we do this with a namelist option instead of code that must be edited by the user?